PCGen-Formula/code/src/java/pcgen/base/formula/parse/SimpleNode.java

```diff
@@ -25,7 +25,6 @@
 	private Node parent;
 	private Node[] children;
 	private int id;
-	private FormulaParser parser;
```

Comment:
This is generated code, I think, so I don't want to alter generated code, since it then leaves us in teh position of having to do a more complicated diff later if we regenerate the code for some reason